src/transformers/file_utils.py

```diff
@@ -2300,16 +2300,14 @@ def get_file_from_repo(
             use_auth_token=use_auth_token,
         )
 
-    except RepositoryNotFoundError as err:
-        logger.error(err)
+    except RepositoryNotFoundError:
```

Comment:
Technically, this instance of `logging.error(err)` has nothing to do with `from_pretrained()` but I decided to remove it for consistency.